src/member/MemberServiceImpl.java

```diff
@@ -40,7 +60,18 @@ public Member login(String loginId, String loginPw) {
         //  비밀번호 확인시는 member.getLoginPw() 로 비밀번호를 받아올 수 있음
         //  오류시 처리: throw new IllegalArgumentException("로그인 실패: ID 또는 비밀번호가 일치하지 않습니다.");
         //  파이팅 !!
-        return null; // 성공시 Member 리턴 아니면 null
+        Member member = memberRepository.findByLoginId(loginId);
+
+        if (member == null) {
+            return null;
+        }
+
+        if (!member.getLoginPw().equals(loginPw)) {
+            return null;
+        }
```

Comment:
여기서도 예외 처리가 들어가면 좋겠어 ~~

```suggestion
        Member member = memberRepository.findByLoginId(loginId);

        if (member == null || !member.getLoginPw().equals(loginPw)) {
            throw new IllegalArgumentException("로그인 실패: ID 또는 비밀번호가 일치하지 않습니다.");
        }
```

이런 느낌???